app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -87,16 +67,37 @@ class Identification extends React.Component {
   getWorkflowValue = () => (this.state.current.isolation ? WORKFLOW_OPTIONS[1] : WORKFLOW_OPTIONS[0]);
 
   handleDateChange(field, date) {
-    let current = this.state.current;
-    let modified = this.state.modified;
     let self = this;
     this.setState(
-      {
-        current: { ...current, patient: { ...current.patient, [field]: date } },
-        modified: { ...modified, patient: { ...modified.patient, [field]: date } },
+      state => {
+        return {
+          current: { ...state.current, patient: { ...state.current.patient, [field]: date } },
+          modified: { ...state.modified, patient: { ...state.modified.patient, [field]: date } },
+        };
       },
       () => {
-        self.props.setEnrollmentState({ ...this.state.modified });
+        // Automatically calculate age field once a date of birth is entered.
+        let age;
+        const dateOfBirth = self.state.current.patient.date_of_birth;
+        // If date is undefined, age will stay undefined (which nulls out the age field)
+        if (dateOfBirth) {
+          age = 0 - moment(dateOfBirth).diff(moment.now(), 'years');
+          console.log('age first:', age);
```

Comment:
remove console statements on 85 and 88?

Reply:
Thought I removed before pushing, removed now.